fsgs/FSGSDirectories.py

```diff
@@ -163,6 +252,7 @@ def get_data_dir(cls):
     @classmethod
     @functools.lru_cache()
     def databases_dir(cls):
+        # XXX: maybe missing cls.get_portable_dir("databases_dir")?
```

Comment:
IMO this method should probably be called `get_databases_dir()`. Also, shouldn't it check for a portable path like the others?